test/models/author_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class AuthorTest < ActiveSupport::TestCase
+
+  test "author has many posts" do
+    author = Author.create(name: "Steve")
+    post1 = Post.create(title: "Test", body: "Luke, at that speed, will you be able to pull out in time?", author_id: author.id)
```

Comment:
Try `author.posts.create` sometime. This way, you don't have to manually set `author_id`.

Also: `author: author` would have worked.